src/components/aeIdentity/aeIdentity.scss

```diff
@@ -33,35 +19,37 @@
   }
 
   .flex-row{
-    position:relative;
     display: flex;
     justify-content: space-between;
   }
 
   .chunked-address {
-    display: flex;
-    flex-wrap: wrap;
     margin-top: 10px;
-    margin-left: -10px;
   }
 
   .chunked-address .chunk {
-    display: inline-block;
-    margin: 5px 0 0 10px;
-    flex-grow: 1;
-    width: calc(100% * (1/3) - 10px - 1px);
-    justify-content: space-between;
-
     font-family: 'Roboto Mono', monospace;
     font-size: 17px;
     font-weight: 500;
     font-style: normal;
-    color: $white;
     line-height: 1.53;
     letter-spacing: 3.1px;
     font-stretch: normal;
     text-align: left;
   }
+
+  .chunked-address .chunk .row {
```

Comment:
Rename row into chunk-row and simply write .chunk-row (without the parents) pls.